src/main/scala/edu/ie3/simona/model/participant2/load/RandomLoadModel.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * © 2024. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.simona.model.participant2.load
+
+import edu.ie3.datamodel.models.input.system.LoadInput
+import edu.ie3.simona.config.RuntimeConfig.LoadRuntimeConfig
+import edu.ie3.simona.model.participant.control.QControl
+import edu.ie3.simona.model.participant2.load.LoadModel.{
+  LoadModelState,
+  LoadModelWithService,
+}
+import edu.ie3.util.scala.quantities.ApparentPower
+import squants.Power
+import squants.energy.{KilowattHours, Watts}
+
+import java.util.UUID
+
+class RandomLoadModel(
+    override val uuid: UUID,
+    override val id: String,
+    override val sRated: ApparentPower,
+    override val cosPhiRated: Double,
+    override val qControl: QControl,
+    override val referenceScalingFactor: Double,
+) extends LoadModel[LoadModelState]
+    with LoadModelWithService
+
+object RandomLoadModel {
+
+  /** The annual energy consumption that the random profile is scaled to.
+    *
+    * It is said in 'Kays - Agent-based simulation environment for improving the
+    * planning of distribution grids', that the Generalized Extreme Value
+    * distribution's parameters are sampled from input data, that is normalized
+    * to 1,000 kWh annual energy consumption. However, due to inaccuracies in
+    * random data reproduction, the sampled values will lead to an average
+    * annual energy consumption of approx. this value. It has been found by
+    * 1,000 evaluations of the year 2019.
+    */
+  private val profileReferenceEnergy = KilowattHours(716.5416966513656)
```

Comment:
This also only works for the examplary random profile (same point as for the standard load model).